packages/schemas/src/types/logto-config.ts

```diff
@@ -54,7 +52,11 @@ export const adminConsoleDataGuard = z.object({
   developmentTenantMigrationNotification: z
     .object({
       isPaidTenant: z.boolean(),
-      tag: z.nativeEnum(TenantTag),
+      /**
```

Comment:
To eliminate misunderstanding, better add comments here to illustrate that this `tag` is for DB rollback use.

Reply:
Updated.